test/Identity.IntegrationTest/Endpoints/IdentityServerTests.cs

```diff
@@ -179,6 +178,255 @@ public async Task TokenEndpoint_GrantTypePassword_WrongAuthEmailHeader_Fails()
         AssertHelper.AssertJsonProperty(root, "error_description", JsonValueKind.String);
     }
 
+    [Theory]
+    [BitAutoData(OrganizationUserType.Owner)]
+    [BitAutoData(OrganizationUserType.Admin)]
+    public async Task TokenEndpoint_GrantTypePassword_WithOwnerOrAdmin_WithSsoPolicyEnabled_WithEnforceSsoPolicyForAllUsersTrue_Throw(OrganizationUserType organizationUserType, int generatedUsername)
+    {
+        var organizationId = new Guid("5e77c2cd-f84b-4a27-affa-12c08d1adbda");
+        var deviceId = "92b9d953-b9b6-4eaf-9d3e-11d57144dfeb";
+        var username = $"{generatedUsername}@example.com";
```

Comment:
Is there any reason these need to be fixed values? Can we use autoData, or move them to static variables, or inside the `CreateOrganizationWithSsoPolicy` method (that's the only place they're used)? Basically looking for a way to clean up the arrange phase, and these are declared in each new test here.